packages/typespec-vscode/src/typespec-utils.ts

```diff
@@ -0,0 +1,82 @@
+import { readdir } from "fs";
+import path, { dirname } from "path";
+import vscode from "vscode";
+import { Executable } from "vscode-languageclient/node.js";
+import { StartFileName } from "./const.js";
+import logger from "./log/logger.js";
+import { isFile, loadModule, normalizeSlash } from "./utils.js";
+
+export const toOutput = (str: string) => {
+  str
+    .trim()
+    .split("\n")
+    .forEach((line) => logger.info(line));
+};
+export const toError = (str: string) => {
+  str
+    .trim()
+    .split("\n")
+    .forEach((line) =>
+      logger.error(line, [], {
+        showOutput: true,
+        showPopup: false,
+      }),
+    );
+};
+
+export async function resolveTypeSpecCli(absolutePath: string): Promise<Executable | undefined> {
```

Comment:
We already have this in tsp-executable-resolver.ts. And you may use the runCliCommand() in tsp-language-client class directly which is easier unless you have any special requirement to run cli command?